environment_py3_linux.yml

```diff
@@ -28,3 +26,15 @@ dependencies:
   - scikit-learn
   - pykdtree
   - zarr
+
+  # Testing
+  - pytest
+  - pytest-html
+  - coverage
+  
+  # Linting
+  - flake8>=2.1.0
+
+  # Docs
+  - sphinx<6
+  
```

Comment:
Strange End of Line (End of File) character here? (Also in the other three files). Do you know what's going on?

Reply:
Its used by GitHub to denote files that don't end with a "\n" character. There was some whitespace at the end of the file left over from my IDE. I've fixed this now.

This is also partly my motivation for #1285 (which I'm using for my local development anyway now).

[[StackOverflow] Why text files should end with newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline), [[StackOverflow] What's the significance of the "No newline at end of file" log?](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)